spec/cucumber/rake/forked_spec.rb

```diff
@@ -14,8 +14,8 @@ module Rake
       context 'when running with bundler' do
         let(:bundler) { true }
 
-        subject { Task::ForkedCucumberRunner.new(
-            libs, binary, cucumber_opts, bundler, feature_files) }
+        subject do Task::ForkedCucumberRunner.new(
+            libs, binary, cucumber_opts, bundler, feature_files) end
```

Comment:
think the end here should be newlined?

Reply:
I can try new lining it, but given my comment on json_spec, it may break things. Technically, it should just be a one-line request, but I think it's on two due to line-length. :grumble:

Reply:
Nevermind, things are fine after changing.